Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	1	CoordinateOverlap
	1

[Sun Feb  7 21:04:37 2021]
rule CoordinateOverlap:
    input: data/gc-scrape.csv, data/presence.df.rds, data/presence.poly.rds
    output: data/relevant-records.csv
    jobid: 0

[Sun Feb  7 21:04:42 2021]
Error in rule CoordinateOverlap:
    jobid: 0
    output: data/relevant-records.csv

RuleException:
CalledProcessError in line 61 of /home/adrian/PhD-Laptop/Pd_Geocache/workflow/snakefile:
Command 'set -euo pipefail;  Rscript --vanilla /home/adrian/PhD-Laptop/Pd_Geocache/workflow/.snakemake/scripts/tmp111aeu4p.coordinate-overlap.R' returned non-zero exit status 1.
  File "/home/adrian/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2189, in run_wrapper
  File "/home/adrian/PhD-Laptop/Pd_Geocache/workflow/snakefile", line 61, in __rule_CoordinateOverlap
  File "/home/adrian/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 529, in _callback
  File "/home/adrian/anaconda3/lib/python3.7/concurrent/futures/thread.py", line 57, in run
  File "/home/adrian/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 515, in cached_or_run
  File "/home/adrian/anaconda3/lib/python3.7/site-packages/snakemake/executors/__init__.py", line 2201, in run_wrapper
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/adrian/PhD-Laptop/Pd_Geocache/workflow/.snakemake/log/2021-02-07T210437.399651.snakemake.log
